packages/profiler/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "cosmwasm-profiler"
+version = "0.0.0"
+authors = ["Tomasz Kurcz <uint@lavabit.com>"]
```

Comment:
I think this code's author is not our in this time. So I think it is not good to change the author. How do you think about it? If we modify something in this code, ant that time, I think we can add our information in the `authors`.

Reply:
I commented because I thought all authors in our repository would be changed since there was a history of batch changes to authors, including packages that only had commits for migration to our repository.(https://github.com/line/cosmwasm/commit/4964bbf8f06e8b5ad05a5cd174a3bf0788c4eba9)

However, if you are going to stipulate as you say about the timing of authors changes, that would be good.

Reply:
Thank you.